controllers/applicationsnapshotenvironmentbinding_controller_test.go

```diff
@@ -2371,7 +2371,7 @@ var _ = Describe("SnapshotEnvironmentBinding controller", func() {
 				return len(createdBinding.Status.GitOpsRepoConditions) > 0 && createdBinding.Status.GitOpsRepoConditions[0].Reason == "GenerateError"
 			}, timeout, interval).Should(BeTrue())
 
-			Expect(createdBinding.Status.GitOpsRepoConditions[0].Message).Should(ContainSubstring("failed to decode devfile json"))
+			Expect(createdBinding.Status.GitOpsRepoConditions[0].Message).Should(ContainSubstring("cannot unmarshal string into Go value of type map[string]interface"))
```

Comment:
What does the full error message look like with this? The test just checks for the substring, so I wasn't sure.

If it's just `cannot unmarshal string into Go value of type map[string]interface`, it's not really user friendly (compared to the old message), and we should probably update the error string that's set in the SEB status. If it's just a subset, and it mentions something along the lines of "failed to decode devfile" or "failed to parse devfile", then that's fine.

Reply:
it looks like,

```
Expected
        <string>: Component updated failed: error parsing devfile because of non-compliant data due to json: cannot unmarshal string into Go value of type map[string]interface {}
    to contain substring
        <string>: cannot unmarslhal string into Go value of type map[string]interface
```

WDYT?

Reply:
Looks good